cogs/ReactForRole.py

```diff
@@ -0,0 +1,716 @@
+#!/usr/bin/env python
+
+"""
+KoalaBot Reaction Roles Code
+
+Commented using reStructuredText (reST)
+"""
+# Futures
+
+# Built-in/Generic Imports
+import re
+from typing import *
+
+import discord
+import emoji
+from discord.ext import commands
+
+# Own modules
+import KoalaBot
+from utils import KoalaDBManager, KoalaColours
+
+# Libs
+
+# Constants
+UNICODE_DISCORD_EMOJI_REGEXP: re.Pattern = re.compile("^:(\w+):$")
+CUSTOM_EMOJI_REGEXP: re.Pattern = re.compile("^<a?:(\w+):(\d+)>$")
+UNICODE_EMOJI_REGEXP: re.Pattern = re.compile(emoji.get_emoji_regexp())
+
+
+class ReactForRole(commands.Cog):
+    def __init__(self, bot: discord.Client):
+        self.bot = bot
+        KoalaBot.database_manager.create_base_tables()
+        KoalaBot.database_manager.insert_extension("ReactForRole", 0, True, True)
+        self.rfr_database_manager = ReactForRoleDBManager(KoalaBot.database_manager)
+        self.rfr_database_manager.create_tables()
+
+    @commands.group(name="reactForRole", aliases=["rfr", "react_for_role", "ReactForRole"])
+    async def react_for_role_group(self, ctx: commands.Context):
+        return
+
+    @commands.check(KoalaBot.is_admin)
+    @react_for_role_group.command(name="createMessage")
```

Comment:
Maybe change to `createMsg` to stay consistent with welcome and other cogs
```suggestion
    @react_for_role_group.command(name="createMsg")
```
Repeat for other "Message" commands

Reply:
Actually may be better as just "create"
```suggestion
    @react_for_role_group.command(name="create")
```